### ISSUE TITLE
1080p > videos are choppy hickup every 5 secs

### ISSUE THREAD

**Issue by Schizo** [brion/ogv.js#552]
Some videos (given in the example) seem to be choppy, every 4-5 seconds there is a little hiccup. To rule out network IO, I've run it on a local server but still the same effect. How can we figure out where the bottle neck is? Is there a profiling doc?

**Comment by brion** [comment]
Could be related to the 1 MiB read chunk size; at higher resolutions you might be running through that data sooner and it's not providing enough of a buffer. Can you repro this with files from the [demo page](https://brionv.com/misc/ogv.js/demo/), or else do you have a publicly-available file to test with?

**Comment by Schizo** [comment]
Yes, I've first reproduced it on the demo-page and then served that file locally and had the same issue. Is there a way to raze the 1 MiB read chunk size? I could see if that would solve the issue.

![image](https://user-images.githubusercontent.com/38006/76369535-1eb46280-62f1-11ea-84cd-491332ac3b01.png)

**Comment by Schizo** [comment]
Any updates on this? Or a hint were I could change the cache size?

**Comment by yafesdev** [comment]
@brion I'm experiencing similar issues, I've tried to change OGVPlayer.js:2467, adding `chunkSize: 16 * 1024 * 1024,` (default is 1 * 1024 * 1024). However, that didn't seem to affect the streamed chunk size. Any Ideas?

**Comment by yafesdev** [comment]
Not sure how the Example  VP8 Video files were encoded but afer reencoding `Tears_of_Steel_in_4k_-_Official_Blender_Foundation_release.webm.1440p.webm` my cpu load wen't down and no more lagging on my notebook.

```
ffmpeg -i ~/Downloads/Tears_of_Steel_in_4k_-_Official_Blender_Foundation_release.webm.1440p.webm  -c:v libvpx -qmin 0 -qmax 50 -b:v 9M  -slices 4 -g 1  ./demo/media/RED1440pReencoded9M.vp8.webm 
```
I just added a high bitrate and -g 1 to have a keyframe on every frame (for quick seeking purposes), I've tested it both in threaded: true/false, and it was playing back fine. On my Iphone though it was very choppy
